sql/catalyst/src/main/antlr4/org/apache/spark/sql/catalyst/parser/SqlBase.g4

```diff
@@ -221,7 +221,7 @@ statement
         multipartIdentifier partitionSpec?                             #loadData
     | TRUNCATE TABLE multipartIdentifier partitionSpec?                #truncateTable
     | MSCK REPAIR TABLE multipartIdentifier                            #repairTable
-    | op=(ADD | LIST) identifier .*?                                   #manageResource
+    | op=(ADD | LIST) identifier (STRING | .*?)                        #manageResource
```

Comment:
should it be `(STRING | .*)?`

Reply:
i used `(STRING | .*?) ` because atleast a file path must be given to ADD/LIST commands.

Reply:
I think (STRING | .*?) should also work fine.

Reply:
yea, but isn't `(STRING | .*)?` more logical? or there are some perf concerns?

Reply:
`(STRING | .*)?` will do greedy matching. `.*` will take any amount of input and thus its made non-greedy by using `.*?` .

Reply:
i see, thanks for the clarification!